Assets/Scripts/UI/UITextOverlay.cs

```diff
@@ -13,7 +13,6 @@ public class UITextOverlay : MonoBehaviour
     private bool isTyping = false;
```

Comment:
Unclear name - does this mean the player is currently typing, or text is scrolling in a dialogue window?
```suggestion
    private bool isTextScrolling = false;
```

Reply:
Oh yea that's super fair! Great suggestion